idealadarsh/README.md

```diff
@@ -0,0 +1,12 @@
+# Simple Flask Application
```

Comment:
Nice!! now you can clean your PR, as you do not require the files of idealadarsh

Reply:
Should I close my PR? Am I right?